warehouse/utils/utils.go

```diff
@@ -135,6 +135,8 @@ const (
 	MinioSecretAccessKey = "secretAccessKey"
 )
 
+var DeprecatedColumnsRegex = regexp.MustCompile(`.*-deprecated-[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$`)
```

Comment:
Is this a UUID in the end ? Also can you write a comment upwards describing what this regex matches ideally ?

Reply:
`abc-deprecated-1234-8321..`